guideforms.py

```diff
@@ -0,0 +1,308 @@
+        required=False, label='',
+        widget=forms.Textarea(
+            attrs={'rows': 2, 'cols': 50, 'placeholder': 'Notes',
+                   'maxlength': 1480})),
+
+    'web_dev_views': forms.ChoiceField(
+        label='Web / Framework developer views',
+        choices=models.WEB_DEV_VIEWS.items(),
+        initial=models.DEV_NO_SIGNALS,
+        help_text='If unsure, default to "No signals".'),
+
+    'web_dev_views_link': forms.URLField(
+        required=False, label='',
+        help_text='Citation link.'),
+
+    'web_dev_views_notes': forms.CharField(
+        required=False, label='',
+        widget=forms.Textarea(
+            attrs={'rows': 2, 'cols': 50, 'placeholder': 'Notes',
+                   'maxlength': 1480}),
+        help_text=('Reference known representative examples of opinion, '
+                   'both positive and negative.')),
+
+    'ergonomics_risks': forms.CharField(
+        label='Ergonomics Risks', required=False,
+        widget=forms.Textarea(attrs={'cols': 50, 'maxlength': 1480}),
+        help_text=
+        ('Are there any other platform APIs this feature will frequently be '
+         'used in tandem with? Could the default usage of this API make it '
+         'hard for Chrome to maintain good performance (i.e. synchronous '
+         'return, must run on a certain thread, guaranteed return timing)?')),
+
+    'activation_risks': forms.CharField(
+        label='Activation Risks', required=False,
+        widget=forms.Textarea(attrs={'cols': 50, 'maxlength': 1480}),
+        help_text=
+        ('Will it be challenging for developers to take advantage of this '
+         'feature immediately, as-is? Would this feature benefit from '
+         'having polyfills, significant documentation and outreach, and/or '
+         'libraries built on top of it to make it easier to use?')),
+
+    'security_risks': forms.CharField(
+        label='Security Risks', required=False,
+        widget=forms.Textarea(attrs={'cols': 50, 'maxlength': 1480}),
+        help_text=
+        ('List any security considerations that were taken into account '
+         'when deigning this feature.')),
+
+    'experiment_goals': forms.CharField(
+        label='Experiment Goals', required=False,
+        widget=forms.Textarea(attrs={'cols': 50, 'maxlength': 1480}),
+        help_text=
+        ('Which pieces of the API surface are you looking to gain insight on? '
+         'What metrics/measurement/feedback will you be using to validate '
+         'designs? Double check that your experiment makes sense given that '
+         'a large developer (e.g. a Google product or Facebook) likely '
+         'can\'t use it in production due to the limits enforced by origin '
+         'trials.\n\nIf Intent to Extend Origin Trial, highlight new/different '
+         'areas for experimentation. Should not be an exact copy of goals '
+         'from the first Intent to Experiment.')),
+
+    'experiment_timeline': forms.CharField(
```

Comment:
Can we break this into separate begin and end fields?

Reply:
I added a TODO for that change.  For this CL, I am just breaking up the existing form into smaller forms.  experiment_timeline was a single field on the old form and in our data.  So, splitting it would require migrating old data or coping with both formats.